EnglishFilter/sections/general_extensions.txt

```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74959
+thefmovies.me#%#//scriptlet("prevent-window-open")
```

Comment:
Where do you need this rule? I don't have popups with other rules.

Reply:
I get with the other rules.

Reply:
@zloyden It's because you are using the Adguard Ad-Blocker Extension. And I am using the AdGuard App Windows.

Reply:
We are using all AdGuards. One of popups is closed, when AdGuard for Windows is enabled.
When `thefmovies.me#%#//scriptlet("prevent-window-open")` is used, `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "zfgloaded")` is unnecessary. Only two rules are required on my end:
```
player.cineflixtv.net #%#//scriptlet("abort-on-property-read", "BetterJsPop")
thefmovies.me#%#//scriptlet("prevent-window-open")
```

Reply:
Ok Sure I will check.

Reply:
Done.